test/orders/service/OrdersService.spec.ts

```diff
@@ -66,6 +66,42 @@ describe('OrdersService', () => {
         const actualRefund1 = actualOrder.refunds[0];
         expect(actualRefund1.channelRefundId).to.equal(expectedRefund.channelRefundId);
         expect(actualRefund1.supplierRefundId).to.equal(expectedRefund.supplierRefundId);
+        // @ts-ignore
+        expect(2).to.equal(expectedRefund.transactions.length);
+        // @ts-ignore
+        const expectedTransaction1 = actualRefund1.transactions[0];
+        // @ts-ignore
+        const actualTransaction1 = actualRefund1.transactions[0];
+        expect(actualTransaction1.id).to.equal(expectedTransaction1.id);
+        expect(actualTransaction1.amount).to.equal(expectedTransaction1.amount);
+        expect(actualTransaction1.message).to.equal(expectedTransaction1.message);
+        expect(actualTransaction1.status).to.equal(expectedTransaction1.status);
+        // @ts-ignore
+        const expectedTransaction2 = actualRefund1.transactions[1];
```

Comment:
Expected should not be pulled from actual. If you look, you are comparing two of the same objects together. Change where you are pulling the expected transaction